docs/sources/reference/cli/run.md

```diff
@@ -137,6 +138,9 @@ When `--cluster.name` is provided, nodes will only join peers who share the same
 By default, the cluster name is empty, and any node that doesn't set the flag can join.
 Attempting to join a cluster with a wrong `--cluster.name` will result in a "failed to join memberlist" error.
 
+The `--cluster.use-discovery-v1` flag can be used to switch back to the older, v1 version of the cluster peer discovery mechanism
+in case of any issues with the newer version. This flag will be deprecated in the future and eventually removed.
```

Comment:
As I add support for A records, I will add here a note about what differences are between these mechanisms in the follow-up PR.